src/sentry/conf/server.py

```diff
@@ -821,6 +822,15 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "sentry.integrations.tasks",
 )
 
+# tmp(michal): Default configuration for post_process* queueus split
```

Comment:
Should this still be considered temporary?

```suggestion
# tmp(michal): Default configuration for post_process* queues split
```

Reply:
Yes, this will be replaced by `CELERY_SPLIT_QUEUE_ROUTES` below. I'd remove the comment when I remove the setting.